intrahost.py

```diff
@@ -531,17 +546,17 @@ def merge_to_vcf(
             raise LookupError(
                 "There must be an isnv file for each sample. %s samples, %s isnv files" % (len(samples), len(isnvs)))
 
-        for fileName in alignments:
-            with util.file.open_or_gzopen(fileName, 'r') as inf:
-                # get two independent iterators into the alignment file
-                number_of_aligned_sequences = count_iter_items(Bio.SeqIO.parse(inf, 'fasta'))
-                # -1 is to account for inclusion of reference in the alignement in addition
-                # to the assemblies
-                if not (number_of_aligned_sequences - 1) == len(isnvs) == len(samples):
-                    raise LookupError(
-                        """samples, isnvs, and alignments must have the same number of elements
-                        (plus an extra reference record in the alignment).
-                        %s does not have the right number of sequences""" % fileName)
+        # for fileName in alignments:
```

Comment:
This commented out section of `intrahost.merge_to_vcf` appears to have been simply a pre-screening sanity check / safety check to make sure the inputs were self consistent. Is this check no longer necessary or is it just that it doesn't function for guessed sample names?

Reply:
I'll adapt it for the both cases, where sample names are provided, and when they are not.